README.rst

```diff
@@ -1,118 +1,70 @@
+#. Save the configuration with ``tutor config save``.
+#. Build the image and launch your platform with ``tutor local launch``.
 
 Compatibility Notes
 --------------------
 
-+-------------------+----------------+
-| Open edX Release  |     Version    |
-+===================+================+
-|      Ironwood     |   < 0.9 < 3.0  |
-+-------------------+----------------+
-|       Juniper     |   >= 0.9 < 3.0 |
-+-------------------+----------------+
-|        Koa        |   >= 2.2 < 5.0 |
-+-------------------+----------------+
-|       Lilac       |   >= 2.2 < 5.0 |
-+-------------------+----------------+
-|       Maple       | >= 4.0 < 6.0   |
-+-------------------+----------------+
-|       Nutmeg      |  >= 5.0 < 6.0  |
-+-------------------+----------------+
-|       Olive       |  >= 5.0 < 6.0  |
-+-------------------+----------------+
-|       Palm        |      >= 6.0    |
-+-------------------+----------------+
-|      Quince       |      >= 7.0    |
-+-------------------+----------------+
-
-The following changes to the plugin settings are necessary. If the release you are looking for is
-not listed, then the accumulation of changes from previous releases is enough.
-
-**Ironwood**
-
-.. code-block:: yaml
-
-    EOX_TAGGING_GET_ENROLLMENT_OBJECT: "eox_tagging.edxapp_wrappers.backends.enrollment_i_v1"
-    EOX_TAGGING_GET_COURSE_OVERVIEW: "eox_tagging.edxapp_wrappers.backends.course_overview_i_v1"
-    EOX_TAGGING_BEARER_AUTHENTICATION: "eox_tagging.edxapp_wrappers.backends.bearer_authentication_i_v1"
-
-**Koa, Lilac, Maple, Nutmeg, Olive, Palm and Quince**
-
-.. code-block:: yaml
-
-    EOX_TAGGING_GET_ENROLLMENT_OBJECT: "eox_tagging.edxapp_wrappers.backends.enrollment_l_v1"
-
-
-Those settings can be changed in ``eox_tagging/settings/common.py`` or, for example, in ansible configurations.
-
-**NOTE**: the current ``common.py`` works with Open edX Lilac version.
++------------------+--------------+
+| Open edX Release | Version      |
++==================+==============+
+| Ironwood         | < 0.9 < 3.0  |
++------------------+--------------+
+| Juniper          | >= 0.9 < 3.0 |
++------------------+--------------+
+| Koa              | >= 2.2 < 5.0 |
++------------------+--------------+
+| Lilac            | >= 2.2 < 5.0 |
++------------------+--------------+
+| Maple            | >= 4.0 < 6.0 |
++------------------+--------------+
+| Nutmeg           | >= 5.0 < 6.0 |
++------------------+--------------+
+| Olive            | >= 5.0 < 6.0 |
++------------------+--------------+
+| Palm             | >= 6.0       |
++------------------+--------------+
+| Quince           | >= 7.0       |
++------------------+--------------+
+
+The plugin is configured for the latest release (Quince). The following changes in the plugin settings should be applied to be used for previous releases.
+If the release you are looking for is not listed, the actual configuration is sufficient or it is incompatible with the current eox-tagging version.
```

Comment:
I think a better word to use here would be "current" instead of "actual"
```suggestion
If the release you are looking for is not listed, then either the current configuration is sufficient, or the release is incompatible with the current eox-tagging version.
```

Reply:
The reason for not using _current_ is because it would be twice in the same sentence, however, I'll accept the change and replace the second _current_ with _latest_.

Reply:
@dcoa Thank you for accepting the suggested change. I just wanted to clarify that "actual" and "current" have different meanings. "Actual" refers to something real or factual, while "current" refers to something that is up to date or happening now. In this context, "current" is the appropriate term to use.

Your idea to use "latest" for the second instance to avoid repetition is a good solution

Reply:
same suggestion as https://github.com/eduNEXT/eox-tenant/pull/210#discussion_r1667756960